sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -579,4 +579,47 @@ export class StudentController extends BaseController {
       hasRestriction: studentRestrictionStatus.hasRestriction,
     } as StudentDetailDTO;
   }
+
+  /**
+   * controller to save the files submitted by the student
+   * with student uploader form.
+   * @Body payload
+   */
+  @AllowAuthorizedParty(AuthorizedParties.student)
+  @Post("upload-files")
```

Comment:
I would say that this method is not actually creating resources, it is updating the resources already present on DB, right?
I would suggest changing it to a PATCH.

Reply:
totally agree